src/main/python/hydra/lib/hydrabase.py

```diff
@@ -46,14 +46,15 @@ class AppGroup(object):
     group_name:     group name
     analyser:       Analyser class "name" e-g HAnalyser not HAnalyser()
     """
-    def __init__(self, hydra, app_name, group_name, analyser=None):
+    def __init__(self, hydra, app_name, group_name, num_app_instances, analyser=None):
```

Comment:
Yes but the argument belongs in the "create_app_xxx" type of call not in the constructor of the class.